.github/set_up_cromwell_action/action.yml

```diff
@@ -3,22 +3,23 @@
 name: 'Set Up Cromwell Steps'
 description: Specific steps that will set up git secrets, java, sbt, and Cromwell on the local machine. 
 inputs:
-  cromwell_repo_token: #As an input to this action, you are required to pass in a token that can be used to authenticate while checking out Cromwell.
+  cromwell_repo_token:
+    description: "As an input to this action, you are required to pass in a token that can be used to authenticate while checking out Cromwell."
```

Comment:
IntelliJ believes that a `description` is required, so I put the comment into one.